fastapi/routing.py

```diff
@@ -528,10 +528,15 @@ def __init__(
             self.response_field = None  # type: ignore
             self.secure_cloned_response_field = None
         self.dependencies = list(dependencies or [])
-        self.description = description or inspect.cleandoc(self.endpoint.__doc__ or "")
-        # if a "form feed" character (page break) is found in the description text,
-        # truncate description text to the content preceding the first "form feed"
-        self.description = self.description.split("\f")[0].strip()
+
+        # Get docstring and handle raw strings
+        docstring = inspect.cleandoc(self.endpoint.__doc__ or "")
+        if docstring.startswith('r"""') or docstring.startswith("r'''"):
```

Comment:
This will not work, `r'''...'''` is already handled by interpreter, and there is no difference between this form, `'''...'''` and `'...'` at all. This could be done only at AST level